apps/website/src/data/found-animal.ts

```diff
@@ -242,6 +242,196 @@ const data: FoundAnimalFlow = {
+                            {
+                              name: "No",
+                              flow: {
+                                prompt: macros.general.rehab(
+                                  "the squirrel",
+                                  "If you are sure the parents are not nearby, and do not return within a few hours",
+                                ),
+                              },
+                            },
+                          ],
+                        },
+                      },
+                      {
+                        name: "No",
+                        flow: {
+                          prompt: macros.general.leave("the squirrel"),
+                        },
+                      },
+                    ],
+                  },
+                },
+              ],
+            },
+          },
+        ],
+      },
+    },
+    {
+      name: "Raccoon",
+      flow: {
+        prompt: ["Does the raccoon appear to be sick or injured?"],
+        options: [
+          {
+            name: "Yes",
+            flow: {
+              prompt: [
+                "Is the raccoon an adult or a baby?",
+                "Baby raccoons are normally between 4 and 14 inches in length, whereas a fully grown adult can be up to 3 feet long. Babies are often extremely vocal, growling, hissing, and snorting if disturbed.",
+              ],
+              options: [
+                {
+                  name: "Adult",
+                  flow: {
+                    prompt: [
+                      "If you have found an injured adult raccoon, although it may be hard to accept, adult raccoons can rarely be caught and successfully treated.",
+                      "If you see an adult raccoon with an injury, leave the animal alone. Even though the injury may take a long time to heal, this is far preferable to the trauma of chase and capture.",
+                    ],
+                  },
+                },
+                {
+                  name: "Baby",
+                  flow: {
+                    prompt: macros.general.rehab("the baby raccoon"),
+                  },
+                },
+              ],
+            },
+          },
+          {
+            name: "No",
+            flow: {
+              prompt: [
+                "Have you found a baby raccoon that's alone with no mother in sight?",
+              ],
+              options: [
+                {
+                  name: "Yes",
+                  flow: {
+                    prompt: [
+                      "Be careful not to create an orphan raccoon accidentally. When a baby raccoon is separated from its mother, it will stay where it is until the mother returns.",
+                      "Monitor the baby from a distance to make sure the mother is still caring for it. Do not attempt to feed or otherwise care for the baby, as this may lead to it imprinting on humans and becoming dependent on them.",
```

Comment:
```suggestion
                      "Monitor the baby from a distance to make sure the mother is still caring for it. Do not attempt to feed or otherwise care for the baby, as this may lead to it becoming dependent on humans.",
```
Imprint is the incorrect word here (my bad).